eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -2125,11 +2133,12 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
   private def handleLocalError(cause: Throwable, d: Data, msg: Option[Any]) = {
     cause match {
       case _: ForcedLocalCommit => log.warning(s"force-closing channel at user request")
+      case _ if stateName == WAIT_FOR_OPEN_CHANNEL => log.info(s"${cause.getMessage} while processing msg=${msg.getOrElse("n/a").getClass.getSimpleName} in state=$stateName")
```

Comment:
This was long overdue. How about putting it in `WARN` though?

Reply:
It's a bit noisy, but you're right, a warning would make more sense and it's easy to filter out these log lines if we don't want to see them.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1817/commits/ee9a1e03f284e3e2250b086cffe369d3d5751017